TKGame/Components/Concrete/Player_GraphicsComponent.cs

```diff
@@ -10,8 +10,10 @@ namespace TKGame.Components.Concrete
 {
     internal class Player_GraphicsComponent : GraphicsComponent
     {
-        void GraphicsComponent.Update(Entity entity, SpriteBatch spriteBatch)
+        void GraphicsComponent.Update(Entity entity)
         {
+            SpriteBatch spriteBatch = TKGame.SpriteBatch;
```

Comment:
Should we choose to keep creating these local `spriteBatch` variables, or should we just replace all instances of `spriteBatch` with `TKGame.SpriteBatch` and not have this local variable?